library/common/extensions/filters/http/platform_bridge/filter.cc

```diff
@@ -487,6 +487,18 @@ void PlatformBridgeFilter::FilterBase::onResume() {
       on_resume_(pending_headers, pending_data, pending_trailers, stream_complete_,
                  parent_.platform_filter_.instance_context);
   if (result.status == kEnvoyFilterResumeStatusStopIteration) {
+    if (result.pending_headers != nullptr) {
+      release_envoy_headers(*result.pending_headers);
+      free(result.pending_headers);
+    }
+    if (result.pending_data != nullptr) {
+      result.pending_data->release(result.pending_data->context);
+      free(result.pending_data);
+    }
+    if (result.pending_trailers != nullptr) {
+      release_envoy_headers(*result.pending_trailers);
+      free(result.pending_trailers);
+    }
```

Comment:
I don't really think these releases should be here - if anything, these should be assertions that the pending entities _are_ nullptr.

Reply:
@goaway, sure. I wasn't sure what the intended behavior was. If the pending entities should be null in this branch, I can switch to assertions.